modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path

```diff
@@ -509,6 +524,16 @@
 	<td>//div/label[@class='control-label' and contains(.,'${key_customFieldName}')]/..//p</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRODUCT_CARD_NAME_AND_PRICE</td>
+	<td>//p[@title='${card_name}']//following-sibling::p[@class='card-text']//span[@class='price-value']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DEFAULT_PRODUCT_IMAGE</td>
+	<td>//div[contains(@style,'default')]</td>
+	<td></td>
+</tr>
```

Comment:
This locator is too generic, please rename it to `PRODUCT_CARD_DEFAULT_IMAGE` and try to identify and use a unique and reusable path(e.g. by using a variable).